criu/cr-dump.c

```diff
@@ -2124,6 +2124,9 @@ int cr_dump_tasks(pid_t pid)
 	int pre_dump_ret = 0;
 	int ret = -1;
 
+	if (kdat.has_madv_guard && !kdat.has_pagemap_scan_guard_pages)
+		pr_warn("ioctl(PAGEMAP_SCAN) doesn't support PAGE_IS_GUARD flag. CRIU dump will crash if dumped process use madvise(MADV_GUARD_INSTALL). Please, consider updating your kernel.\n");
```

Comment:
I'd have this a a function in, say, `kerndat.c` and call it there when detecting `MADV_GUARD _INSTALL` and from `cr-check.c`

Reply:
Great idea! Have done. Thanks ;-)